spec/ApiClient/BlogClientSpec.php

```diff
@@ -37,7 +38,7 @@ public function it_gets_an_article()
     public function it_lists_articles()
     {
         $request = new Request('GET', 'blog-articles?page=1&per-page=20&order=desc&subject[]=cell-biology',
-            ['X-Foo' => 'bar', 'Accept' => 'application/vnd.elife.blog-article-list+json; version=2']);
+            ['X-Foo' => 'bar', 'Accept' => 'application/vnd.elife.blog-article-list+json; version=2', 'User-Agent' => 'eLifeApiClient/'.Version::get()]);
```

Comment:
Separately I've been tempted to remove PHPSpec and replace these with plain unit tests. We can refactor later.